soroban-env-host/src/storage.rs

```diff
@@ -242,6 +242,46 @@ impl Storage {
         self.put_opt(key, Some(val), budget)
     }
 
+    pub fn bump(
+        &mut self,
+        key: &Rc<LedgerKey>,
+        min_ledgers_to_live: u32,
+        ledger_num: u32,
+        budget: &Budget,
+    ) -> Result<(), HostError> {
+        // A expiration ledger bump is considered to be a read only operation. When
+        // concurrent Soroban transactions are added conflicts will be reconciled upstream.
+        let ty = AccessType::ReadOnly;
+        match self.mode {
+            FootprintMode::Recording(_) => {
+                self.footprint.record_access(key, ty, budget)?;
+            }
+            FootprintMode::Enforcing => {
+                self.footprint.enforce_access(key, ty, budget)?;
+            }
+        };
+
+        let mut current = (*self.get(&key, budget)?).clone();
```

Comment:
another unmetered clone here, need to meter or use entry API or something

Reply:
Updated to use metered_clone.